src/ValidatedNumerics.jl

```diff
@@ -62,8 +70,13 @@ end
 
 include("misc.jl")
 include("intervals/intervals.jl")
+
+a..b = @interval(a, b)
+
+include("multidim/multidim.jl")
 include("root_finding/root_finding.jl")
 
+Base.isfinite(x::Interval) = isfinite(x.lo) && isfinite(x.hi)
```

Comment:
I think this is not necessary here, since you are including this function in `src/intervals/special_intervals.jl`.